plugins/ruby-foreman-tasks/debian/control

```diff
@@ -2,14 +2,14 @@ Source: ruby-foreman-tasks
 Section: ruby
 Priority: extra
 Maintainer: Michael Moll <mmoll@mmoll.at>
-Build-Depends: debhelper, foreman (>= 1.17.0~rc1), foreman-sqlite3 (>= 1.17.0~rc1), foreman-assets
+Build-Depends: debhelper, foreman (>= 1.25.0~rc1), foreman-sqlite3 (>= 1.25.0~rc1), foreman-assets
```

Comment:
Wrong file? :)

Reply:
yes :-) I'll fix that once 3.0.1 goes out, we need to fix a spec there to accept tasks 1.0.x